lib/circuitry/config/subscriber_settings.rb

```diff
@@ -0,0 +1,32 @@
+require 'virtus'
+require 'circuitry/config/shared_settings'
+
+module Circuitry
+  module Config
+    class SubscriberSettings
+      include Virtus::Model
```

Comment:
You can then remove the `include Virtus::Model` line here and in publisher_settings